src/ZenstruckFoundryBundle.php

```diff
@@ -114,6 +114,28 @@ public function configure(DefinitionConfigurator $definition): void
                                     ->defaultValue(AbstractORMPersistenceStrategy::RESET_MODE_SCHEMA)
                                     ->values([AbstractORMPersistenceStrategy::RESET_MODE_SCHEMA, AbstractORMPersistenceStrategy::RESET_MODE_MIGRATE])
                                 ->end()
+                                ->arrayNode(AbstractORMPersistenceStrategy::RESET_MODE_MIGRATE)
+                                    ->addDefaultsIfNotSet()
+                                    ->children()
+                                        ->arrayNode('configurations')
+                                            ->info('Migration configurations')
```

Comment:
maybe we could add a little bit more hints?
```suggestion
                                            ->info('Migration configuration files (relative to project directory)')
```
not sure what's the best? relative to project dir? absolute? relative to configuration file?

Reply:
I changed to `Migration configuration files` since we won’t check if paths are absolute or not; is that okay?